functions/src/Controller/Committee.ts

```diff
@@ -13,7 +13,7 @@ export function getCommitteeCollection() {
 export const addCommitteeController = functions.https.onRequest((request, response) => {
 	const committee: Committee = new Committee(request.body);
 
-	addCommittee(committee);
+	void addCommittee(committee);
```

Comment:
You don't need the void here u can just put addCommittee(). the void is a bit unnecessaryy